### ISSUE TITLE
We have entered prices inclusive taxes. We have different tax rates for different states and countries.

### ISSUE THREAD

**Issue by pranaytech** [woocommerce/woocommerce#16541]
For example, when the country is India and state is 
Maharashtra, the tax rate is 12 %. When the Country is India and the state is Gujarat, the tax rate is 9%. When the country is USA, the tax rate is 15%.

What we need is that the total cost of the item including tax should not change according to the tax slab. The cost of the item (exclusive of tax) can change but the cost of item with tax should not change across different states / countries. 

For Example - Item price (inclusive tax) - INR 100
1. Then for Maharashtra (where tax rate is 12%) -The cost of item should be 88 and tax should be 12. Total with tax = 100
2. For Gujarat (where tax rate is 9%) - Cost of item should be 91 and tax should be 9. Total with tax = 100
3. For USA (tax rate is 15%) - Cost of item should be 85 and tax should be 15. Total with tax = 100

Basically the total price (inclusive tax) should not change irrespective of the tax rate. Is this possible in woo commerce ? If so, could you guide us how to implement the same ?

**Comment by mikejolley** [comment]
https://github.com/woocommerce/woocommerce/wiki/How-Taxes-Work-in-WooCommerce#prices-including-tax---experimental-behavior
